reference/versions/0.5.0/reference.json

```diff
@@ -0,0 +1,112 @@
+{
```

Comment:
That's my fault: I started using `AnalysisGraph` from reference within workflow/factory but I shouldn't have. We need an alternative within `./lib` to `getChildNodesNames` from a definition so we don't need to use anything from `./reference`.